src/components/Formik/Forms/UserViewForm.tsx

```diff
@@ -0,0 +1,120 @@
+import React,{useState} from "react";
+import {Form,Formik,FormikHelpers} from "formik";
+import * as Yup from "yup";
+import {OneStudentResponse} from 'types'
+
+import staticText from "../../../languages/en.pl";
+import {SubmitBtn} from "../../common/SubmitBtn/SubmitBtn";
+import {UserPersonalDataInputs} from "../../UserPersonalDataInputs/UserPersonalDataInputs";
+import {UserEmploymentDataInputs} from "../../UserEmploymentDataInputs/UserEmploymentDataInputs";
+import {UserAboutMeInputs} from "../../UserAboutMeInputs/UserAboutMeInputs";
+import {UserExperienceInputs} from "../../UserExperienceInputs/UserExperienceInputs";
+import {expectedContractTypesValues,expectedTypeWorkValues} from "../../../utils/enumKeys/enumKeys";
+
+
+export interface userFullData{
+    firstName:string;
+    lastName:string;
+    email:string;
+    tel:string;
+    githubUsername:string;
+    portfolioUrls:string;
```

Comment:
type for portfolioUrls will be string[]

Reply:
Nope, here you need to see that I made reduce on portfolio URL, same as on project URL to receive list splitter by enter like requested in specification

UserFullData is other type to OneStudentResponse, see line 40 where I make changes necessary for FE but not in BE response